job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -6,7 +6,7 @@
 import org.kie.jenkins.jobdsl.Constants
 
 def javadk=Constants.JDK_TOOL
-def AGENT_LABEL="kie-rhel7 && kie-mem4g"
+def AGENT_LABEL="kie-rhel && kie-mem512m"
```

Comment:
I don't think we should move this to a constant since it is only used here (very special case)

Reply:
Yes, I meant mainly the other cases.